doc/source/amqp1d.rst

```diff
@@ -0,0 +1,37 @@
+===================
+Amqp1d Daemon Usage
```

Comment:
`AMQP 1.0 Daemon Usage` sounds better since if you set "amqp1d daemon" it sounds like "amqp1 daemon daemon".

Reply:
done